data/resources/ui/pod/creation-page.ui

```diff
@@ -7,180 +7,350 @@
+                                      <object class="GtkBox">
+                                        <property name="orientation">vertical</property>
+                                        <property name="spacing">18</property>
+
+                                            <child>
+                                              <object class="AdwPreferencesGroup">
+
+                                                <child>
+                                                  <object class="AdwActionRow" id="disable_infra_row">
+                                                    <property name="activatable_widget">disable_infra_switch</property>
+                                                    <property name="title" translatable="yes">Disable Infra</property>
+                                                    <child>
+                                                      <object class="GtkSwitch" id="disable_infra_switch">
+                                                        <property name="valign">center</property>
+                                                        <property name="action-name">pod.toggle-infra</property>
+                                                      </object>
+                                                    </child>
+                                                  </object>
+                                                </child>
+
+                                              </object>
+                                            </child>
+
+                                            <child>
+                                              <object class="GtkBox" id="infra_settings_box">
+                                                <property name="orientation">vertical</property>
+                                                <property name="spacing">18</property>
+
+                                                <child>
+                                                  <object class="AdwPreferencesGroup">
+
+                                                    <child>
+                                                      <object class="AdwEntryRow" id="infra_name_entry_row">
+                                                        <property name="activates-default">True</property>
+                                                        <property name="title" translatable="yes">Name</property>
+                                                      </object>
+                                                    </child>
+
+                                                    <child>
+                                                      <object class="PdsPropertyRow" id="infra_local_image_property_row">
+                                                        <property name="key" translatable="yes">Local image</property>
+                                                      </object>
+                                                    </child>
```

Comment:
This is not needed in this case.
It's there in the container creation page, because one could pass an image directly into the container creation page. For example, if you click "Create Container..." in the 3-dots menu of an image row. In this case, the remote search and local selection are disabled.
```suggestion
```